### ISSUE TITLE
[주간회의] 10월 5일 

### ISSUE THREAD

**Issue by unizard** [kangdain/TLO#9]
한주간의 업무를 기록합니다.

**Comment by kangdain** [comment]
### 한 일

- 학습 rate, Overfitting, 일반화 
- 딥러닝의기본 XOR문제와  Back-propagation 에 대한 학습
- 제 3 차 청년 TLO 교육 - 제 1 강 지식재산일반 (월) 
- Linear Regression의 Hypothesis 와 Cost 설명,
- Multi-variable linear regression (화) 
-  Vue.js component의 사용법 학습 (목) 	
- Tensorflow 실습 - Logistic classification , softmax classification, 프로젝트 셋팅 (금)
